application/client.core/src/app/environment/layout/contextmenu/component.ts

```diff
@@ -34,13 +34,15 @@ export class LayoutContextMenuComponent implements OnDestroy, AfterViewInit {
     }
 
     public ngOnDestroy() {
+        ContextMenuService.close = undefined;
```

Comment:
Not a clean solution. I mean it isn't clear how it works and why it's done in this way. See comments bellow